src/test/cypress/integration/ProgrammingExerciseParticipation.spec.ts

```diff
@@ -61,13 +53,9 @@ describe('Programming exercise participations', () => {
             courseManagement.addStudentToCourse(course.id, users.getStudentOne().username);
             courseManagement.addStudentToCourse(course.id, users.getStudentTwo().username);
             courseManagement.addStudentToCourse(course.id, users.getStudentThree().username);
-            courseManagement
-                .createProgrammingExercise({ course })
-                .its('body')
-                .then((exercise) => {
-                    expect(exercise).to.not.be.null;
-                    programmingExercise = exercise;
-                });
+            courseManagement.createProgrammingExercise({ course }).then((exerciseResponse) => {
+                exercise = exerciseResponse.body;
```

Comment:
I don't see the value of the assertion here. The backend shouldn't answer with null in the first place if the request was successfully executed. If it unexpectedly returns null or no body the test will fail anyways because it needs the exercise object. If I add the assertion again it should probably be `to.be.ok` to cover the theoretical possibility that the backend returns null as body